src/lib/pubkey/ec_group/ec_group.cpp

```diff
@@ -631,7 +631,9 @@ EC_Point EC_Group::hash_to_curve(std::string_view hash_fn,
       throw Not_Implemented("EC_Group::hash_to_curve not available for this curve type");
    }
 
-   return hash_to_curve_sswu(*this, hash_fn, input, input_len, domain_sep, domain_sep_len, random_oracle);
+   std::span input_span(input, input_len);
+   std::span dst_span(domain_sep, domain_sep_len);
+   return hash_to_curve_sswu(*this, hash_fn, input_span, dst_span, random_oracle);
```

Comment:
For the record (it's debatable what form is better), this would work as well and is more compact:

```suggestion
   return hash_to_curve_sswu(*this, hash_fn, {input, input_len}, {domain_sep, domain_sep_len}, random_oracle);
```